corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -438,18 +438,18 @@ def _diff_ledgers(self, case_ids):
             )
 
     def _check_for_migration_restrictions(self, domain_name):
+        msgs = []
         if not should_use_sql_backend(domain_name):
-            msg = "does not have SQL backend enabled"
-        elif COUCH_SQL_MIGRATION_BLACKLIST.enabled(domain_name, NAMESPACE_DOMAIN):
-            msg = "is blacklisted"
-        elif any(custom_report_domain == domain_name
-                 for custom_report_domain in settings.DOMAIN_MODULE_MAP.keys()):
-            msg = "has custom reports"
-        elif REMINDERS_MIGRATION_IN_PROGRESS.enabled(domain_name):
-            msg = "has reminders migration in progress"
-        else:
-            return
-        raise MigrationRestricted("{} {}".format(domain_name, msg))
+            msgs.append("does not have SQL backend enabled")
+        if COUCH_SQL_MIGRATION_BLACKLIST.enabled(domain_name, NAMESPACE_DOMAIN):
+            msgs.append("is blacklisted")
+        if any(custom_report_domain == domain_name
+               for custom_report_domain in settings.DOMAIN_MODULE_MAP.keys()):
+            msgs.append("has custom reports")
+        if REMINDERS_MIGRATION_IN_PROGRESS.enabled(domain_name):
```

Comment:
I thought this was one of the last things Giovanni finished before he left? could we get rid of this flag?